internal/trigger/http/rest.go

```diff
@@ -58,7 +58,7 @@ func (trigger *Trigger) requestHandler(writer http.ResponseWriter, r *http.Reque
 	decoder := json.NewDecoder(r.Body)
 	err := decoder.Decode(&event)
 	if err != nil {
-		trigger.logging.Debug("HTTP Body not an Edgex Event")
+		trigger.logging.Error("HTTP Body not an Edgex Event")
```

Comment:
I dont agree with this being an error. Error indicates that we've got an error on the server side due to some internal issue (i.e status code 500). But this is based on what the client sends, which is why i'm returning a 400 Bad Request. In other words, You don't want the logs filling up with this message just because a client doesn't send the correct thing.

Reply:
K, changed back to Debug.